aiida/restapi/common/utils.py

```diff
@@ -270,9 +261,9 @@ def validate_request(
                 'requested (i.e. the path must contain '
                 '/page/)'
             )
-        # 4. No querystring if query type = schema'
-        if query_type in ('schema') and is_querystring_defined:
-            raise RestInputValidationError('schema requests do not allow specifying a query string')
+        # 4. No querystring if query type = projectable_properties'
+        if query_type in ('projectable_properties') and is_querystring_defined:
```

Comment:
This (used to be and) is wrong.
Just the brackets for a single element are ignored and are not converted to a tuple. 
You are just checking is the substring is in there.

For instance
```python
a = 'pro'
b = 'projectable_properties'

a in ('projectable_properties') # True
b in ('projectable_properties') # True
```

You instead want to do
```python
a in ('projectable_properties', ) # False
b in ('projectable_properties', ) # True
```
(note the comma, making it a tuple)

Also, could you please check if the same mistake happens elsewhere?

Reply:
thanks, fixed it.